sequencing/ATACSeq/jobSubmission/0_setUp.sh

```diff
@@ -0,0 +1,174 @@
+## - execute from scripts directory                                                                          ||
+##                                                                                                           ||
+## INPUTS:                                                                                                   || 
+## $1 -> <project name> name of project to set up pipeline for.                                              || 
+##                                                                                                           ||
+## DESCRIPTION: This script sets up the required directories, libraries and modules to run the ATAC-seq      || 
+## pipeline. Will check the config file for the project and the initial paths required for the first steps of|| 
+## the analysis.                                                                                             ||
+##                                                                                                           ||
+## REQUIRES:                                                                                                 ||
+## - File in ${METADIR}/samples.txt that lists sample names.                                                 ||
+## ==========================================================================================================##
+
+
+## ============ ##
+##    SET UP    ##
+## ============ ##
+
+## print start date and time
+echo Job started on:
+date -u
+
+
+LOG_DIR=ATACSeq/logFiles/${USER}/${SLURM_JOB_ID}
+echo "Log files will be moved to dir: " $LOG_DIR
+mkdir -p $LOG_DIR
+mv ATACSetUpS0-${SLURM_JOB_ID}* $LOG_DIR
+
+## ============ ##
+##    CHECKS    ##
+## ============ ##
+
+## CONFIG FILE ##
+
+## Check if config file for input project exist
+
+if [[ -s "/lustre/projects/Research_Project-MRC190311/ATACSeq/$1/config.txt" ]]
+then
+  source "/lustre/projects/Research_Project-MRC190311/ATACSeq/$1/config.txt"
+  echo "Config file for $1 project found."
+else 
+  { echo "Config file for chosen project does not exist or is in the wrong directory. Please create this first." ; exit 1; }  
+fi
+
+## Check if R config file for input project exist
+
+if [[ -s "/lustre/projects/Research_Project-MRC190311/ATACSeq/$1/config.r" ]]
+then
+  echo "R config file for $1 project found."
+else 
+  { echo "R config file for chosen project does not exist or is in the wrong directory. Please create this first." ; exit 1; }  
+fi
+
+## DIRECTORIES ##
+
+## Check that the metadata directory exists and contains sample list.
+if [[ ! -d ${METADIR} ]]
+then
+	echo 'METADATA directory does not exist'
+	exit 1
+elif  test -f ${METADIR}/samples.txt; then
+	mapfile -t SAMPLEIDS < ${METADIR}/samples.txt 
+	echo "Sample list found. Number of sample IDs found:" """${#SAMPLEIDS[@]}"""
+else
+	echo 'samples.txt file does not exist in METADIR'
+fi
+
+## Check that the sampleSheet exist
+if [[ -d ${METADIR} ]] && [[ -s "${METADIR}/sampleSheet.csv" ]]
```

Comment:
This first test of -d ${METADIR} has already been tested for on line 73. So isn't doing anything